src/fvOptions/actuatorLineSource/actuatorLineElement/actuatorLineElement.C

```diff
@@ -499,7 +504,7 @@ void Foam::fv::actuatorLineElement::writePerf()
                 << forceVector_.y() << "," << forceVector_.z() << ","
                 << endEffectFactor_ << "," << tangentialRefCoefficient() << ","
                 << normalRefCoefficient() << "," << tangentialRefForce() << ","
-                << normalRefForce() << endl;
+                << normalRefForce() << "," << dict_.lookup("pitch") << endl;
```

Comment:
This seems to already have been looked up, no?

```suggestion
                << normalRefForce() << "," pitch << endl;
```